release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -37,8 +37,62 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
+xml x1 = xml `<ele>item</ele><!--comment-->text<?foo?>`;
+xml<xml:Element> x2 = xml `<ele>item1</ele><ele>item2</ele>`;
+xml<xml:Comment> x3 = xml `<!--comment1--><!--comment2-->`;
+xml<xml:ProcessingInstruction> x4 = xml `<?foo?><?bar?>`;
+string interpolation = "Interpolation";
+xml<xml:Text> x5 = xml `text ${interpolation}`;
+```
+
+##### Normalization of XML Attribute Values
+In XML template literals, character references within XML attribute values are now normalized to the referring character. For example, the attribute value `e.attr` in the following example is now `a&b`.
+
+```ballerina
+xml:Element e = xml `<elem attr="a&amp;b" />`;
+```
+
+##### Breaking Changes
+- The `lang.array:lastIndexOf` and `lang.array:indexOf` lang library functions can only be used with arrays of which the element types are subtypes of the `anydata` type.
+- A compilation error occurs if the field visibility qualifiers in the including type and the included type with object type inclusion are not the same.
+- Remote method call action cannot be used to call non-remote methods.
+- The `xml` and `readonly` type-descriptors can no longer be used as expressions. A reference to a type definition defined with such type-descriptors needs to be used instead.
+- The argument passed to the second parameter of the `attach` method of the listener for service declarations has been fixed as follows. If there is no attach point in the service `()` (nil) is passed as the argument. If the attach point is the root resource path (“/”) an empty array is passed.
+- The `lang.xml:getContent()` function no longer accepts `xml:Text` as an argument. The new `lang.xml:data()` function can be used instead.
+- `lang.value` functions cannot be called on expressions of type `object`. Function call syntax (`value:func(obj)`) can be used instead.
+- The `lang.stream:reduce()` lang library function will only return the reduced value or an error (if the completion type includes error). It no longer returns a nil `()` value. The return type of `lang.stream:reduce()` has been updated accordingly.
+- An issue where the complement of numeric literals returned incorrect values has been fixed.
+- An issue that allowed negation with numeric literals has been fixed. Negation can be used only with `boolean` values.
+- Subtyping rules have been fixed to consider the `never` type, which denotes an empty set of values, as a subtype of any type `T`.
+- Relational expressions with lists have been fixed to consider the relative order of lists with different sizes.
+- An issue where a non-error value did not match the wildcard match pattern at runtime when the matched expression’s static type included error has been fixed.
+- An issue with Isolation analysis where transfer out validation did not happen when `self` was used in a return statement of an isolated object method has been fixed.
+- An issue where the method qualifiers were now being set for a `readonly` and `class` intersection has been fixed.
```

Comment:
```suggestion
- The issue of the method qualifiers being set for a `readonly` and `class` intersection has been fixed.
```

Reply:
Similar to comment https://github.com/ballerina-platform/ballerina-release/pull/159#discussion_r616726472

Reply:
```suggestion
- An issue where the method qualifiers were not being set for a `readonly` and `class` intersection has been fixed.
```

Reply:
```suggestion
- In a `readonly` and `class` intersection, the method qualifiers are now preserved from the class.
```